### PR TITLE
reset kustomize installition link

### PR DESCRIPTION
**Pull request by ycyaoxdu**
Signed-off-by: ycyaoxdu <yaoyuchen0626@163.com>

seems that kustomize installation link is changed, repalce it with new link

### CODE REVIEW

File: content/en/getting-started/core/cluster-manager.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The are two ways to install the core control plane of open cluster management th
 
 ## Prerequisite
 
-Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/en/getting-started/core/cluster-manager.md:16]:
```suggestion
Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/en/getting-started/core/register-cluster.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ After the cluster manager is installed on the hub cluster, you need to install t
 
 ## Prerequisite
 
-Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/en/getting-started/core/register-cluster.md:16]:
```suggestion
Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/en/getting-started/integration/app-lifecycle.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ For more details, visit the [multicloud-operators-subscription GitHub page](http
 
 You must meet the following prerequisites to install the application lifecycle management add-on:
 
-- Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+- Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/en/getting-started/integration/app-lifecycle.md:26]:
```suggestion
- Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/en/getting-started/integration/policy-controllers.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ After policy framework is installed, you can install the policy controllers to t
 You must meet the following prerequisites to install the policy controllers:
 
 - Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and
-  [`kustomize`](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+  [`kustomize`](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/en/getting-started/integration/policy-controllers.md:19]:
```suggestion
  [`kustomize`](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/en/getting-started/integration/policy-framework.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ various security and configuration aspects to help IT administrators meet their
 You must meet the following prerequisites to install the policy framework:
 
 - Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and
-  [`kustomize`](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+  [`kustomize`](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/en/getting-started/integration/policy-framework.md:29]:
```suggestion
  [`kustomize`](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/zh/getting-started/core/cluster-manager.md [hunk 6]
```diff
@@ -13,7 +13,7 @@ The are two ways to install the core control plane of open cluster management th
 
 ## Prerequisite
 
-Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/zh/getting-started/core/cluster-manager.md:16]:
```suggestion
Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/zh/getting-started/core/register-cluster.md [hunk 7]
```diff
@@ -13,7 +13,7 @@ After the cluster manager is installed on the hub cluster, you need to install t
 
 ## Prerequisite
 
-Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/zh/getting-started/core/register-cluster.md:16]:
```suggestion
Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/zh/getting-started/integration/app-lifecycle.md [hunk 8]
```diff
@@ -23,7 +23,7 @@ For more details, visit the [multicloud-operators-subscription GitHub page](http
 
 You must meet the following prerequisites to install the application lifecycle management add-on:
 
-- Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+- Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/zh/getting-started/integration/app-lifecycle.md:26]:
```suggestion
- Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/zh/getting-started/integration/policy-controllers.md [hunk 9]
```diff
@@ -15,7 +15,7 @@ After policy framework is installed, you can install the policy controllers to t
 
 You must meet the following prerequisites to install the policy controllers:
 
-* Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [`kustomize`](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+* Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [`kustomize`](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/zh/getting-started/integration/policy-controllers.md:18]:
```suggestion
* Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [`kustomize`](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

File: content/zh/getting-started/integration/policy-framework.md [hunk 10]
```diff
@@ -23,7 +23,7 @@ The policy framework provides governance capability to gain visibility, and driv
 
 You must meet the following prerequisites to install the policy framework:
 
-- Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [`kustomize`](https://kubernetes-sigs.github.io/kustomize/installation) are installed.
+- Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [`kustomize`](https://kubectl.docs.kubernetes.io/installation/) are installed.
```
Comment (mikeshng) [inline content/zh/getting-started/integration/policy-framework.md:26]:
```suggestion
- Ensure [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl) and [`kustomize`](https://kubectl.docs.kubernetes.io/installation/kustomize/) are installed.
```

**Review by mikeshng — COMMENTED** [review]
I think the kustomize link should be `https://kubectl.docs.kubernetes.io/installation/kustomize/` instead of `https://kubectl.docs.kubernetes.io/installation/` 

Could you check?

**Comment by ycyaoxdu** [comment]
> I think the kustomize link should be `https://kubectl.docs.kubernetes.io/installation/kustomize/` instead of `https://kubectl.docs.kubernetes.io/installation/`
> 
> Could you check?

sorry for my careless, and thanks a lot!

**Comment by mikeshng** [comment]
/lgtm

/approve

thanks for the fix!

**Pull request merged**